test/test_transforms.py

```diff
@@ -1311,7 +1311,8 @@ def test_random_choice():
             transforms.Resize(15),
             transforms.Resize(20),
             transforms.CenterCrop(10)
-        ]
+        ],
+        [1 / 3, 1 / 3, 1 / 3]
```

Comment:
Is this modification necessary? I thought that the default argument (`None`) should make things compatible?

Reply:
Yes it's not necessary. It's just to show how this would work if we wanted to pass probabilities. If `None` is passed (which is the default argument) it assumes automatically equal probs.